### PR TITLE
PR: auditd loop

### PR DESCRIPTION
**Pull request by mikerenfro**
Cleaned up the commits into a new branch for this PR.

Travis CI builds are failing due to not being able to restart the auditd service. Not sure what the workaround is for that.

### CODE REVIEW

**Comment by watsonb** [comment]
I've merged this PR into a local copy of RHEL7-STIG and applied it to an out-of-the-box RHEL7 server image and it worked without error.

File: tasks/audit_system_call.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+# tasks/audit_system_call.yml
+
+- name: "MEDIUM | RHEL-07-{{ item.id }} | {{ item.create | ternary('PATCH', 'REVERT') }} | All uses of the {% if item.call is defined %}{{ item.call }}{% elif item.path is defined %}{{item.path | basename}}{% endif %} command must be audited."
```
[the remark this answers is not in the document]
Comment (mikerenfro) [reply tasks/audit_system_call.yml]: Added handler for auditd, but Travis builds still fail on the service restart.

**Comment by shepdelacreme** [comment]
@mikerenfro Yeah its probably the container missing something like augenrules or something. I'll need to test it locally to see.

**Comment by shepdelacreme** [comment]
Ok nevermind...apparently auditd won't run properly in a container until they have finished making the daemon namespace aware. Also it looks like you can't restart auditd directly with the systemctl command...so I think the restart handler will need to be changed and we will need to set it so it skips the handler during travis tests. I'm still testing locally.

**Comment by mikerenfro** [comment]
@shepdelacreme When I ran into not being able to restart auditd via `systemctl`, that's why I fell back to calling `service` directly. Thought I'd also tried forcing the Ansible service task to use the service-style backend with [the `use` parameter](http://docs.ansible.com/ansible/latest/service_module.html#options), but either that didn't work, or I had another unrelated problem.

Will defer to whatever sensible alternative you come up with.

**Comment by shepdelacreme** [comment]
@mikerenfro Ok I've got a fix in. I'm going to push my two commits to your pr branch and then we can merge this.

**Comment by mikerenfro** [comment]
@shepdelacreme cool. I assume you're not waiting on anything from my end, right?

**Comment by shepdelacreme** [comment]
@mikerenfro Nope was just diving home while the tests ran! I'll merge now if everything looks good to you.

**Comment by mikerenfro** [comment]
Looks good to you? Then it looks good to me.

**Comment by shepdelacreme** [comment]
Thanks for all this work @mikerenfro!!

**Comment by jamescassell** [comment]
Thanks for all the work on this!

**Pull request merged**